webapp/app.py

```diff
@@ -536,11 +552,15 @@ def format_error(detail, status):
 
         return res
 
+    middlewares = []
+    if GZIP_RESPONSE_ENABLE:
+        middlewares.append(gzip_middleware)
+    middlewares.extend([error_handler, timing_middleware])
```

Comment:
can you initialize on line 555?

```middlewares = [error_handler, timing_middleware]```

or does order of middlewares matter?

Reply:
unfortunately, order matters